documentation/announcing-az-module.md

```diff
@@ -0,0 +1,163 @@
+# Announcing New Module 'Az'
+In August 2018 we released a new module, 'Az' which combines the functionality of the AzureRM and AzureRM.Netcore modules.  Az runs on both PowerShell 5.1 and PowerShell Core.  'Az' ensures that the PowerShell and PowerShell core cmdlets for managing Azure resources will always be in sync and up to date.  In addition, Az will simplify and regularize the naming of Azure cmdlets, and the organization of Azure modules.  Az is intended as a replacement for the AzureRM.Netcore and AzureRM modules.
+
+Az currently ships in CloudShell, and can be found on the PowerShell Gallery [here](https://www.powershellgallery.com/packages/Az/0.2.2)
+
+Az is a new module, and reorganizing and simplifying cmdlet names involves breaking changes, so we have [added features to Az to make it easier to transition to the simplified, normalized names in your existing scripts](#migrating-from-azurerm). 
+
+## New Features
+  - PowerShell and PowerShell Core support in the same module
+  - PowerShell Core Edition and PowerShell Desktop Edition cmdlets always in sync and up to date with latest Azure capabilities
+  - Shortened and Normalized cmdlet names - all cmdlets use the noun 'Az'
+  - Normalized module Organization - data plane and management plane cmdlets in the same module for each service
+  - Enhanced authentication for Netcore
+  - Enable/Disable-AzureRmAlias cmdlets to manage compatibility with scripts based on AzureRM
+
+## Suppoted Platforms
+  - PowerShell 5.1 with .Net Framework 4.7.2 or later [Windows only]
+  - PowerShell Core 6.0 - Windows, linux, MacOS
+  - PowerShell Core 6.1 - Windows, linux, MacOS
+
+## Timeline
+  - Initial Release - August 2018
+  - Az at functional parity with AzureRM - November 2018
+  - Last version of AzureRM with new Azure features - December 2018
+  
+
+## AzureRM Module Support
+Azure RM will continue to be supported, and important bugs will be fixed, but new development and new Azure capabilities will be shipped only in Az starting December 2018
+
+## Migrating From AzureRM
+
+To make it easier for existing scripts to migrate to AzureRM, we have provided cmdlets to create aliases that map the cmdlet names in AzureRM into the appropriate cmdlets in Az.  When the aliases are enabled, any script that uses AzureRM cmdlet names should run against Az without modification.
```

Comment:
@markcowl "...for existing scripts to migrate _from_ AzureRM, ..." 

or,

"...for existing scripts to migrate to _Az_, ..."